README.md

```diff
@@ -103,6 +103,26 @@ The full process is described graphically in the [workflow diagram](./docs/workf
 
    > **NOTE**: ⏳ `pip install` may take some time, depending on your internet connection.
 
+   See [the GPU acceleration documentation](./docs/gpu-acceleration.md) for how to
+   to enable hardware acceleration for inference and training on AMD ROCm,
+   NVidia CUDA, and Apple Metal Performance Shaders (MPS).
+
+   ```shell
+   rm -rf venv
+   python3 -m venv venv
+   source venv/bin/activate
+   (venv) $ pip cache remove llama_cpp_python
+   (venv) $ pip install git+https://github.com/instructlab/instructlab.git@stable -C cmake.args="-DLLAMA_CUBLAS=on"
+   ```
+
+   ```shell
+   rm -rf venv
+   python3 -m venv venv
+   source venv/bin/activate
+   (venv) $ pip cache remove llama_cpp_python
+   (venv) $ pip install git+https://github.com/instructlab/instructlab.git@stable -C cmake.args="-DLLAMA_METAL=on"
+   ```
```

Comment:
note the need to do s/fixme// ... i had to get it to stop ending my suggestion early

Reply:
I have removed the `rm -rf` line and instead used `python -m venv --clear venv`

```shell
$ python -m venv --help
usage: venv [-h] [--system-site-packages] [--symlinks | --copies] [--clear] [--upgrade] [--without-pip] [--prompt PROMPT] [--upgrade-deps]
            ENV_DIR [ENV_DIR ...]

Creates virtual Python environments in one or more target directories.

positional arguments:
  ENV_DIR               A directory to create the environment in.

options:
  -h, --help            show this help message and exit
  --system-site-packages
                        Give the virtual environment access to the system site-packages dir.
  --symlinks            Try to use symlinks rather than copies, when symlinks are not the default for the platform.
  --copies              Try to use copies rather than symlinks, even when symlinks are the default for the platform.
  --clear               Delete the contents of the environment directory if it already exists, before environment creation.
  --upgrade             Upgrade the environment directory to use this version of Python, assuming Python has been upgraded in-place.
  --without-pip         Skips installing or upgrading pip in the virtual environment (pip is bootstrapped by default)
  --prompt PROMPT       Provides an alternative prompt prefix for this environment.
  --upgrade-deps        Upgrade core dependencies (pip) to the latest version in PyPI

```

Reply:
But the instructions still read as telling people to install and then clear and install second time with the correct options. Part of what my suggestion attempted to do was guide people into running a single code block of install instructions.

Reply:
Makes sense, thanks for the suggestion, @russellb 

I have pushed another commit that refactors the README and addresses a bunch of other issues. For example I had to include `--upgrade-deps`, so we get a recent pip version on RHEL 9.